docs/team/index.md

```diff
@@ -68,6 +68,18 @@ user you want to transfer the App to, and hit **Transfer Ownership**.
 ## Team Account
 If you want to share Apps, Dashboards, etc. with others, or use Sematext as a team or company, we recommend creating one account that is not tied to any individual's email.  Instead, if you have a group/team email such as dev@... or engineering@... or monitoring@... or even sematext@... at your organization, we suggest you sign up with that email and then invite to that account everyone else who needs to be able to access Apps, Dashboards, Alert Rules, etc. in that account.  This way all assets created in Sematext remain in this team account and when individuals leave the company their access to this account can easily be removed with just a few clicks. 
 
+Changing the ownership of the account can be done with the "Convert to Team Account" feature. By converting your account to a Team Account everything in your account will be transferred to the Team Account - Apps, Alert Rules, Dashboards, Payment Method, etc.
```

Comment:
Let's add the hyperlinked EU inside parenthesis

Reply:
A hyperlink to the conversion page?

Reply:
Yes, as we do elsewhere in the docs